### PR TITLE
add libpam-google-authenticator

### PR DESCRIPTION
**Pull request by demhademha**

### CODE REVIEW

**Review by sunflsks — APPROVED** [review]

File: libpam-google-authenticator.mk [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+endif
+
+SUBPROJECTS   += libpam-google-authenticator
+LIBPAM-GOOGLE-AUTHENTICATOR_COMMIT := 0b02aadc28ac261b6c7f5785d2f7f36b3e199d97
+LIBPAM-GOOGLE-AUTHENTICATOR_VERSION := 0~20210222.$(shell echo $(LIBPAM-GOOGLE-AUTHENTICATOR_COMMIT) | cut -c -7)
+DEB_LIBPAM-GOOGLE-AUTHENTICATOR_V   := $(LIBPAM-GOOGLE-AUTHENTICATOR_VERSION)-1
+
+libpam-google-authenticator-setup: setup
+	-[ ! -e "$(BUILD_SOURCE)/libpam-google-authenticator-$(LIBPAM-GOOGLE-AUTHENTICATOR_COMMIT).tar.gz" ] \
+		&& wget -nc -O$(BUILD_SOURCE)/libpam-google-authenticator-$(LIBPAM-GOOGLE-AUTHENTICATOR_COMMIT).tar.gz \
```
Comment (1Conan) [inline libpam-google-authenticator.mk:12]: missing -q flag

File: libpam-google-authenticator.mk [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+ifneq ($(wildcard $(BUILD_WORK)/libpam-google-authenticator/.build_complete),)
+libpam-google-authenticator:
+	@echo "Using previously built libpam-google-authenticator."
+else
+libpam-google-authenticator: libpam-google-authenticator-setup
+	cd $(BUILD_WORK)/libpam-google-authenticator && ./bootstrap.sh && ./configure -C \
+		--build=$$($(BUILD_MISC)/config.guess) \
+		--host=$(GNU_HOST_TRIPLE) \
+		--prefix=$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)
+			+$(MAKE) -C $(BUILD_WORK)/libpam-google-authenticator
```
Comment (1Conan) [inline libpam-google-authenticator.mk:25]: indentation

File: libpam-google-authenticator.mk [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+	+$(MAKE) -C $(BUILD_WORK)/libpam-google-authenticator install \
+		DESTDIR="$(BUILD_STAGE)/libpam-google-authenticator"
+		touch $(BUILD_WORK)/libpam-google-authenticator/.build_complete
```
Comment (1Conan) [inline libpam-google-authenticator.mk:28]: indentation

File: libpam-google-authenticator.mk [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+endif
+
+libpam-google-authenticator-package: libpam-google-authenticator-stage
+	# libpam-google-authenticator.mk Package Structure
+	rm -rf $(BUILD_DIST)/libpam-google-authenticator
+	mkdir -p $(BUILD_DIST)/libpam-google-authenticator
+
+	# libpam-google-authenticator.mk Prep libpam-google-authenticator
+	cp -a $(BUILD_STAGE)/libpam-google-authenticator $(BUILD_DIST)
+	mv -f $(BUILD_DIST)/libpam-google-authenticator$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/lib/security $(BUILD_DIST)/libpam-google-authenticator$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/lib/pam
```
Comment (1Conan) [inline libpam-google-authenticator.mk:38]: missing `/`s

**Review by 1Conan — CHANGES_REQUESTED** [review]

**Pull request merged**